src/apps/vpn/telemetry/metrics.yaml

```diff
@@ -256,7 +256,7 @@ session:
   daemon_session_source:
     type: string
     description: |
-      (Mobile-only) Where the session started from. Should be "app", "system settings",
+      (Mobile-only) Where the session started from. Should be "app", "system",
       or "autostart on boot".
```

Comment:
How about "autostart on boot" for Android?

Reply:
Good call, done.

Reply:
I think I actually want to remove “autostart on boot” from a potential description. (If I recall my research from a month ago, it was nearly impossible to figure out the difference between “autostart on boot” and “started from system settings” programatically. It may have been possible with a lot of grossness, but with “anything outside the app” being the important piece, it seems like the “system”/“app” split is the most important piece here.)